app/utils/repo-actions-item-component-mixin.js

```diff
@@ -21,18 +25,36 @@ export default Ember.Mixin.create({
     return this.get('item.canRestart') && this.get('userHasPermissionForRepo');
   }.property('userHasPermissionForRepo', 'item.canRestart'),
 
+  displayFlashError(status) {
+    let type = this.get('type');
+    if (status === 422 || status === 400) {
+      this.get('flashes').error(`This ${type} can't be cancelled`);
```

Comment:
This message will not work for restarting, we should either get back to do it inline or parametrise it somehow.